setup.py

```diff
@@ -38,15 +38,6 @@
     authors = SPARQLWrapper.__authors__
     url = SPARQLWrapper.__url__
 
-
```

Comment:
the whole sense behind this snippet was to avoid duplication of libraries in both `setup.py` and `requirements.txt`